db/src/bootstrap.rs

```diff
@@ -33,63 +34,63 @@ use values;
 pub const TX0: i64 = 0x10000000;
 
 lazy_static! {
-    static ref V1_IDENTS: Vec<(&'static str, i64)> = {
-        vec![(":db/ident",             entids::DB_IDENT),
-             (":db.part/db",           entids::DB_PART_DB),
-             (":db/txInstant",         entids::DB_TX_INSTANT),
-             (":db.install/partition", entids::DB_INSTALL_PARTITION),
-             (":db.install/valueType", entids::DB_INSTALL_VALUETYPE),
-             (":db.install/attribute", entids::DB_INSTALL_ATTRIBUTE),
-             (":db/valueType",         entids::DB_VALUE_TYPE),
-             (":db/cardinality",       entids::DB_CARDINALITY),
-             (":db/unique",            entids::DB_UNIQUE),
-             (":db/isComponent",       entids::DB_IS_COMPONENT),
-             (":db/index",             entids::DB_INDEX),
-             (":db/fulltext",          entids::DB_FULLTEXT),
-             (":db/noHistory",         entids::DB_NO_HISTORY),
-             (":db/add",               entids::DB_ADD),
-             (":db/retract",           entids::DB_RETRACT),
-             (":db.part/user",         entids::DB_PART_USER),
-             (":db.part/tx",           entids::DB_PART_TX),
-             (":db/excise",            entids::DB_EXCISE),
-             (":db.excise/attrs",      entids::DB_EXCISE_ATTRS),
-             (":db.excise/beforeT",    entids::DB_EXCISE_BEFORE_T),
-             (":db.excise/before",     entids::DB_EXCISE_BEFORE),
-             (":db.alter/attribute",   entids::DB_ALTER_ATTRIBUTE),
-             (":db.type/ref",          entids::DB_TYPE_REF),
-             (":db.type/keyword",      entids::DB_TYPE_KEYWORD),
-             (":db.type/long",         entids::DB_TYPE_LONG),
-             (":db.type/double",       entids::DB_TYPE_DOUBLE),
-             (":db.type/string",       entids::DB_TYPE_STRING),
-             (":db.type/boolean",      entids::DB_TYPE_BOOLEAN),
-             (":db.type/instant",      entids::DB_TYPE_INSTANT),
-             (":db.type/bytes",        entids::DB_TYPE_BYTES),
-             (":db.cardinality/one",   entids::DB_CARDINALITY_ONE),
-             (":db.cardinality/many",  entids::DB_CARDINALITY_MANY),
-             (":db.unique/value",      entids::DB_UNIQUE_VALUE),
-             (":db.unique/identity",   entids::DB_UNIQUE_IDENTITY),
-             (":db/doc",               entids::DB_DOC),
+    static ref V1_IDENTS: Vec<(symbols::NamespacedKeyword, i64)> = {
+        vec![(ns_keyword!("db", "ident"),             entids::DB_IDENT),
```

Comment:
You introduced a nice macro for doing the splitting in the previous patch, and then you don't use that nice form in this patch.  I don't think you can do the string checking at compile time with a macro (you probably can with a build script), so is this by design?  If it is, get rid of the earlier patch entirely.